pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -58,6 +59,7 @@ stdenv.mkDerivation rec {
 
   runtimeDependencies = [
     cups.lib
+    pango.out
```

Comment:
use `(lib.getLib pango)` is a cleaner approach, and you may change `cups.lib` to this style by the way.